.travis.yml

```diff
@@ -1,5 +1,9 @@
 language: java
 
+script:
+  - jdk_switcher use oraclejdk8
+  - mvn install javadoc:aggregate
```

Comment:
This defines the targets to run... we always wish to build the Javadoc target as this lets us know if the Javadoc is JDK8 compliant.